src/CliArguments.hs

```diff
@@ -60,6 +63,10 @@ parser = do
         if watch
           then True
           else warn
+    , optimize =
+        if optimize
```

Comment:
would be nice to use a custom type for this. I should have done that for the other flags as well.